lib/ex_phone_number/metadata/phone_number_description.ex

```diff
@@ -9,28 +9,66 @@ defmodule ExPhoneNumber.Metadata.PhoneNumberDescription do
   def from_xpath_node(nil), do: nil
   def from_xpath_node(xpath_node) do
     kwlist =
-      xpath_node |> xmap(
-        national_number_pattern: ~x"./nationalNumberPattern/text()"o |> transform_by(&normalize_pattern/1),
-        possible_number_pattern: ~x"./possibleNumberPattern/text()"o |> transform_by(&normalize_pattern/1),
-        example_number: ~x"./exampleNumber/text()"o |> transform_by(&normalize_string/1)
-      )
-    struct(%PhoneNumberDescription{}, kwlist)
+      xpath_node
+      |> xmap(
```

Comment:
cosmetic refactor:

The pipe operator is not needed here.

```suggestion
      xmap(
        xpath_node,
        national_number_pattern: ~x"./nationalNumberPattern/text()"o |> transform_by(&normalize_pattern/1),
        national_possible_lengths: ~x"./possibleLengths/@national"o |> transform_by(&normalize_range/1),
        local_possible_lengths: ~x"./possibleLengths/@localOnly"o |> transform_by(&normalize_range/1),
        example_number: ~x"./exampleNumber/text()"o |> transform_by(&normalize_string/1)
      )
```